sycl/plugins/cuda/pi_cuda.cpp

```diff
@@ -1514,6 +1514,10 @@ pi_result cuda_piDeviceGetInfo(pi_device device, pi_device_info param_name,
   case PI_DEVICE_INFO_GPU_SUBSLICES_PER_SLICE:
   case PI_DEVICE_INFO_GPU_EU_COUNT_PER_SUBSLICE:
   case PI_DEVICE_INFO_MAX_MEM_BANDWIDTH:
+    // TODO: Check if Intel device UUID extension is utilized for CUDA.
```

Comment:
If so, shouldn't we make it `oneapi` extension instead of `intel`?

Reply:
I am not sure if that was the intension of the jira, but I could be wrong. @smaslov-intel what do you think we should do?

Reply:
I think we should keep it to be intel extension for now.